lib/main/atomts.ts

```diff
@@ -28,6 +28,11 @@ export function activate(state: State) {
   pluginManager = new PluginManager(state)
 
   setImmediate(() => handlePromise(checkAndInstallDependencies()))
+  // add warning for the slow down
+  if (atom.config.get("atom-typescript.checkAllFilesOnSave")) {
```

Comment:
This option is disabled by default. I can't get behind annoying the user with pop-ups in this case.

Reply:
I wanted it to be a one-time update release note thing, but I could not find a way to do so.

Maybe something like this: https://github.com/JunoLab/atom-julia-client/blob/master/lib/package/release-note.js